### DISCUSSION TITLE
Export more than one less than all

### DISCUSSION THREAD

**Discussion by eirlon** [pavelkomarov/exportify#69] [Ideas]
Hey, would it be possible to implement a way to select multiple playlists to export at once? I have a lot of saved playlist and naturally I just want to download the ones I've created but not even all of those anyways. I have folders with the playlists I want to download so a way to choose a playlist folder would also be helpful but really just want to be able to select multiple playlists at once but not all the playlists. thanks!

**Comment by pavelkomarov** [comment]
It’s probably possible if there is a way to fetch user playlist folders
somehow, so I can figure out what’s in each. But I’m probably not going to
implement this. Seems niche. I’ve never heard of folders before.

**Comment by eirlon** [comment]
sorry if im misunderstanding but just clarifying. you said you'd probably not implement the folders situation. does that mean you also will not implement a way to select more than one playlist at a time in the current set up?

**Reply by pavelkomarov** [reply]
I think selecting a bunch and then hitting download is the same as clicking the download buttons for several in succession. That is unless clicking a subsequent button interrupts an ongoing download. I'm actually not sure whether it would.

**Reply by pavelkomarov** [reply]
I just tried, and it doesn't interrupt ongoing downloads. Good news.

**Reply by singhkhan498-png** [reply]
You can try MusConv — it lets you select multiple playlists to transfer at once, so you don’t have to move everything. It works with Spotify, YouTube Music, Apple Music, and more.

**Comment by pavelkomarov** [comment]
My browser settings are such that all my downloads just go to the desktop or to the downloads folder, and I’m not prompted to give a filename and path each time. That would be annoying. See if that can help. Clicking in rapid succession becomes way easier when there isn’t a dialog popup.

In the case you don’t want playlists by other authors, and those are a minority, you’d want a blacklist instead of a whitelist.
